tutor4points/core/templates/home.html

```diff
@@ -1,10 +1,72 @@
 <!--Dashboard Page-->
 {% extends 'base.html' %}
 
+{% block extrahead%}
+{% load static %}
+<link rel="stylesheet" type="text/css" href="{% static 'css/home.css' %}" />
+{% endblock%}
+
 {% block heading %}
 Dashboard
 {% endblock %}
 
 {% block content %}
+{% with user=request.user %}
+<div class="container-fluid main-container">
+    <div class="row d-flex">
+        <div class="col">
+            <div class="dashboard-container d-flex flex-row">
+                <h2>Account Balance: {{ user.total_points }} points</h3>
+            </div>
+            <div class="dashboard-container">
+                <div class="list-container">
+                    <h2>Tutor Requests</h2>
+                </div>
+            </div>
+        </div>
+        <div class="col">
+            <form class="form-inline search-box">
+                <input class="form-control search-field" type="search" placeholder="Search subjects/classes">
+                <button class="btn btn-outline-success search-button" type="submit">Search</button>
+            </form>
+            <div class="dashboard-container">
+                <div class="list-container">
+                    <h2>Suggested Sessions</h2>
+                        <!-- TODO: Refactor repeated code -->
+                        {% for tutor in tutors %}
```

Comment:
The tutors do not appear on the "suggested sessions" page. You might need to pass in the tutors from one of the views or create a view to pass in the data.

Reply:
Oh my changes to the view file didn't appear in the commit. I'll update that and put it in.